.github/workflows/build.yml

```diff
@@ -10,12 +15,14 @@ jobs:
 
     steps:
       - name: Checkout 🛎
-        uses: actions/checkout@v4
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          persist-credentials: false
 
       - name: Setup Platform 🛠
-        uses: oven-sh/setup-bun@v1
+        uses: oven-sh/setup-bun@f4d14e03ff726c06358e5557344e1da148b56cf7 # v1.2.2
         with:
-          bun-version: latest
+          bun-version: 1.1.13
```

Comment:
I forgot to note this as well: For a 1.0 piece of software, Bun is _very_ unstable. I always pin to the patch to prevent breakage. I found my builds very flaky otherwise.

Reply:
Thank you for pinning this as well, it makes sense!